NEWS.md

```diff
@@ -6,6 +6,8 @@
   
 * fixed bug in `fct_collapse()` so it now correctly collapses factors when `group_other = TRUE` (#172), and makes `"Other"` the last level (#202) (@gtm19, #172 & #202)
 
+* Updated calls to **rlang** package (#233). Package called globally with `@import rlang` instead of `rlang::` calls. Removed all `rlang::` calls (@gralgomez, @gabrielodom, #233)
```

Comment:
For future reference, you don't need a news bullet here since it doesn't affect the user of the package.

Reply:
Gotcha! Thanks